py3_analysis/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM socrata/python3
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y python-numpy python-scipy
```

Comment:
Each line in a docker file creates a new filesystem layer. The layer based on `apt-get update` isn't a particularly meaningful one, so these two commands are best combined on a single line, e..g.
`RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y python-numpy python-scipy`